packages/diagnostic/.eslintignore

```diff
@@ -0,0 +1,7 @@
+*.config.js
```

Comment:
confirm this aligns with the latest PR's which "optimize" various configs and consume from global

Reply:
I will optimize linter configs later